docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -2507,6 +2741,82 @@
     "tab.prod(numeric_only=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "c87d4f95",
+   "metadata": {},
+   "source": [
+    "### Table.count()\n",
+    "\n",
+    "```\n",
+    "Table.count(axis=0, numeric_only=False)\n",
+    "```\n",
+    "\n",
+    "Returns the count of non-NA values across the given axis.\n",
```

Comment:
Probably worth changing this to non null rather than non-NA as given q has better support for null values it will be more accurate

Reply:
This change makes sense.